concepts.wip/arithmetic-operators/introduction.md

```diff
@@ -3,3 +3,26 @@
 Julia provides a number of [arithmetic operators](https://en.wikipedia.org/wiki/Arithmetic#Arithmetic_operations) for numeric types[^1].
 
 [^1]: You can find a list of them in the [Julia Manual](https://docs.julialang.org/en/v1/manual/mathematical-operations/#Arithmetic-Operators).
+
+Many of the arithmetic operations are straightforward. Below are examples of some operations in Julia that may have different **syntax** in other languages, ex: Python.
```

Comment:
Two notes:

- It's often better to avoid language that calls something "straightforward", "easy" etc. to not discourage people who may not find it as straightforward. (even if in this particular case that seems unlikely)
- A comparison with a specific language will likely only be useful for a small subset of people. Perhaps we can keep the spirit of this sentence but change it to something like "Below are examples of some arithmetic operations in Julia. You may find that Julia embraces unicode operators more than many other languages."

Reply:
Applied change as per suggestion.